devops/staging/values.yaml

```diff
@@ -36,7 +37,7 @@ iam-cache-server-helm:
   # runAsUser: 1000
 
   service:
-    type: NodePort
+    type: ClusterIP
```

Comment:
@Timtech4u @vickenliu any concerns with this `service` `type` change? Will this work for the ssi-hub staging deployment?

Reply:
i think so,